res/skins/LateNight/style_palesun.qss

```diff
@@ -0,0 +1,3119 @@
+/* LateNight 2.3 // PaleSun theme
+
+* containers
+  * colors, borders, background tiles
+  * layout spacing
+
+* colors & icons
+  * font colors
+  * button styles
+  * button icons
+
+* library
+* menu & special widget styles
+
+
+/**********************************************************************
+************** container background colors & borders *****************/
+#SkinContainer {
+  background-color: #f7f7f7;
+}
+
+#213k1, #213kCompact1, #213kMini1,
```

Comment:
There are some strange replacements which basically eliminate a lot of styles:
`#Deck` > `#213k`
`#Beat` > `#415t`